hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BPServiceActor.java

```diff
@@ -816,7 +816,7 @@ void register(NamespaceInfo nsInfo) throws IOException {
     // off disk - so update the bpRegistration object from that info
     DatanodeRegistration newBpRegistration = bpos.createRegistration();
 
-    LOG.info(this + " beginning handshake with NN");
+    LOG.info("{} beginning handshake with NN:{}", this, nnAddr);
```

Comment:
Please keep the same format of log, such as delete the blank at end of a sentence, leave one blank between two words. such as,
L819: `LOG.info("{} beginning handshake with NN: {}", this, nnAddr);`
L831: `LOG.info("Problem connecting to server: {}", nnAddr);`

Reply:
@Hexiaoqiao Thanks for your help reviewing the code, I will fix it.